app/views/cards/_form.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= simple_form_for @card, html: { class: 'form-horizontal' } do |f| %>
+  <%= f.input :original_text, label: 'Текст' %>
+  <%= f.input :translated_text, label: 'Перевод' %>
+  <%= f.input :review_date, label: 'Дата ревью' %>
```

Comment:
why you need this, if you create review date **automatically**

Reply:
i create default value, may be user wants to set another date, cause it's difficult word for him